app/crud/labels.py

```diff
@@ -214,3 +224,22 @@ def verify_time_fields(cls, time: datetime.datetime) -> datetime.datetime:
     def enabled_or_dynamic(cls, b: bool) -> bool:
         assert isinstance(b, bool)
         return b
+
+
+class PredefinedLabel(Label):
+    @classmethod
+    def validate_all(cls, session: snowflake.snowpark.Session):
```

Comment:
can you create a ticket to move the predefined label def'n and creation out of sql and into this?

Reply:
Of course